README.md

```diff
@@ -13,23 +13,37 @@
 
```

Comment:
`.coverage` should probably be removed from the PR and added as an entry in `.gitignore`?

Reply:
Fixed here:
https://github.com/nautobot/nautobot-plugin-data-validation-engine/pull/38/files#

`.gitignore` already contains `.coverage`